velox/functions/sparksql/Arithmetic.h

```diff
@@ -25,6 +25,43 @@
 
 namespace facebook::velox::functions::sparksql {
 
+template <typename T>
+struct RoundFunction {
```

Comment:
@zhejiangxiaomai Doesn't this need to be registered ? Wont calling round in eval call the previous default implementation ?

Reply:
Yes, I will fix it.

Reply:
https://github.com/facebookincubator/velox/blob/main/velox/functions/sparksql/RegisterArithmetic.cpp#L49-L57 I found round were registered.

Reply:
Can you make sure that it calls your implementation and not presto's https://github.com/facebookincubator/velox/blob/4b653edf45cf4dafec8ef7d79e1a39063a0e6dce/velox/functions/prestosql/Arithmetic.h#L136 .

Reply:
Thanks, I just add namespace prefix `sparksql::`.